hildon/hildon-gtk.h

```diff
@@ -66,6 +66,18 @@ typedef enum {
    HILDON_SIZE_THUMB_HEIGHT     = 2 << 2, /* set to thumb height */
    HILDON_SIZE_AUTO             = (HILDON_SIZE_AUTO_WIDTH | HILDON_SIZE_AUTO_HEIGHT)
 } HildonSizeType;
+
+// Support for placeholder texts is used extensively around libhildon,
+// but requires a patched version of Gtk+. It was added upstream in Gtk 3.2.
+// Therefore, for a non-Maemo version of Gtk+ we expose a dummy function.
```

Comment:
Please try to keep the style of the package you're making changes to.
This package have comments of the form /* */ in most cases.

Reply:
Done.